ios/Classes/FlutterWebviewPlugin.m

```diff
@@ -79,17 +80,18 @@ - (void)initWebview:(FlutterMethodCall*)call {
         rc = self.viewController.view.bounds;
     }
     
-    self.webview = [[UIWebView alloc] initWithFrame:rc];
-    self.webview.delegate = self;
-    
+    self.webview = [[WKWebView alloc] initWithFrame:rc];
+    self.webview.navigationDelegate = self;
+    [self.viewController.view addSubview:self.webview];
+
     if (withZoom != (id)[NSNull null] && [withZoom boolValue]) {
-        self.webview.scalesPageToFit = YES;
+        self.webview.scrollView.delegate = self;
+        _enableZoom = [withZoom boolValue];
     }
     
     if (hidden != (id)[NSNull null] && [hidden boolValue]) {
```

Comment:
so can we remove the condition ?

Reply:
🤔 Yes! BOOL values are 0 by default in obj-c, so we don't need to check if it's nil